solana-programs/reward-manager/program/src/processor.rs

```diff
@@ -480,6 +480,7 @@ impl Processor {
         bot_oracle_info: &AccountInfo<'a>,
         payer_info: &AccountInfo<'a>,
         rent_info: &AccountInfo<'a>,
+        mint_info: Option<&AccountInfo<'a>>,
```

Comment:
Do we like the optional arg? could also maybe have the "reward_token_recipient" be the mint in the case of the burn?